client/request.go

```diff
@@ -57,6 +58,10 @@ func (c *Client) doRequest(req *http.Request, v interface{}) error {
 		errorResponse.RawMessage = responseBody
 		return errorResponse
 	}
+	if resp.StatusCode == 204 {
```

Comment:
I'd put a TODO here saying that it's hopefully temporary and because the Vercel is incorrectly using 204 when they should be using 404.